rgw/v2/tests/aws/test_aws_non_ascii.py

```diff
@@ -0,0 +1,125 @@
+"""
+Usage: test_aws_non_ascii.py -c <input_yaml>
+
+<input_yaml>
+    Note: Following yaml can be used
+    configs/test_aws_non_ascii.yaml
+
+Operation:
+Do a S3 Put request with the object body having a non ascii character
+"""
+
+
+import argparse
+import json
+import logging
+import os
+import random
+import sys
+import traceback
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+
+
+from v2.lib import resource_op
+from v2.lib.aws import auth as aws_auth
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.tests.aws import reusable as aws_reusable
+from v2.tests.s3_swift import reusable as s3_reusable
+from v2.utils import utils
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+log = logging.getLogger(__name__)
+TEST_DATA_PATH = None
+
+
+def test_exec(config, ssh_con):
+    """
+    Executes test based on configuration passed
+    Args:
+        config(object): Test configuration
+    """
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    user_info = resource_op.create_users(no_of_users_to_create=config.user_count)
+
+    for user in user_info:
+        user_name = user["user_id"]
+        log.info(user_name)
+        endpoint = aws_reusable.get_endpoint(ssh_con)
+        aws_auth.do_auth_aws(user)
+
+        for bc in range(config.bucket_count):
+            bucket_name = utils.gen_bucket_name_from_userid(user_name, rand_no=bc)
+            aws_reusable.create_bucket(bucket_name, endpoint)
+            log.info(f"Bucket {bucket_name} created")
+            object_name = "ˍ´--øÆ.txt"
```

Comment:
Hi @TejasC88 , IMO instead of making new script file,shall we make use of existing script file:
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/tests/aws/test_aws.py#L73
post line 73 please include condition:

i.e: if config.test_ops.get("non_ascii_character_upload", False):

and place below snipet:
object_name = "ˍ´--øÆ.txt
 utils.exec_shell_cmd(f"fallocate -l 1K {object_name}")
 aws_reusable.put_object(bucket_name, object_name, endpoint)
 log.info(
  "Object name and body containing non ascii character upload succeeded"  )

Reply:
@ckulal Done , pls review

Reply:
updated pass log: http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/non_ascii.txt